commit a0d63c1a1fc673f094bb309635e98ab4c11354e9
Author: Sage Weil <sweil@redhat.com>
Date:   Mon Apr 29 16:27:58 2019 -0500

    mgr/DaemonServer: Fix pgp_num throttling min
    
    This was meant to limit the size of any single change to 1/2 of the
    max_misplaced (see the comment above!).
    
    Signed-off-by: Sage Weil <sage@redhat.com>

diff --git a/src/mgr/DaemonServer.cc b/src/mgr/DaemonServer.cc
index a12c093fcb9..1b7f0ce1f08 100644
--- a/src/mgr/DaemonServer.cc
+++ b/src/mgr/DaemonServer.cc
@@ -2568,7 +2568,7 @@ void DaemonServer::adjust_pgs()
 	    } else {
 	      double room =
 		std::min<double>(max_misplaced - misplaced_ratio,
-				 misplaced_ratio / 2.0);
+				 max_misplaced / 2.0);
 	      unsigned estmax = std::max<unsigned>(
 		(double)p.get_pg_num() * room, 1u);
 	      int delta = target - p.get_pgp_num();
